src/rs.js

```diff
@@ -0,0 +1,132 @@
+// 1. Character Frequency.
+
+/**
+ * Calculates the character frequencies in a string. Whitespace and
```

Comment:
@bmathews that explains where the file I thought I had saved to recruiting went! good catch :facepalm: